package.json

```diff
@@ -19,5 +19,13 @@
         "^0.58.0": "consumeStatusBar"
       }
     }
+  },
+  "providedServices": {
+    "vim-mode": {
+      "description": "",
+      "versions": {
+        "0.33.0": "provideVimMode"
```

Comment:
In general, we don't intent to have these service versions match the package versions, since the package may go through many versions w/o changing its API. I believe we made this mistake in the status-bar service, which made for a confusing example.

Would you mind changing this to `0.1.0`? I think that is a good default to use, since the service is still new and should be considered somewhat unstable/evolving.